test/integrations/destinations/zoho/router/account.ts

```diff
@@ -0,0 +1,142 @@
+import { upsertPayload1V2, upsertPayload2V2, commonConnectionConfigV2 } from '../../zoho/common';
+import { defaultMockFns } from '../../zoho/mocks';
+
+const commonDestinaitonConfig = {
+  DestinationDefinition: {
+    Config: {
+      cdkV2Enabled: true,
+    },
+  },
+  deliveryAccount: {
+    options: {
+      region: 'EU',
+    },
+  },
+  Config: {
+    module: 'Leads',
```

Comment:
this config is no longer valid as these come from connConfig now, remove non-required fields